src/manual/task-execution/provisioning.md

```diff
@@ -0,0 +1,46 @@
+---
+filename: task-execution/provisioning.md
+title: Provisioning
+order: 40
+---
+
+In the simplest case, a worker implementation is installed and configured
+manually on a host and executes tasks indefinitely. For most purposes, it is
+far more cost-effective to dynamically adjust the pool of available workers
+based on the work to be done. This process is called provisioning.
+
+A provisioner operates by monitoring the pool of running workers and the
+corresponding queues. Based on the queue length and the configuration of the
+provisioner, it may create new workers or signal to running workers that they
+should stop. Of course, provisioners might take into account additional
```

Comment:
Yep, I'm trying to separate a general "what is provisioning" section from a specific "how does the AWS provisioner do it" section (without getting into too much detail in the latter - that can be added in the reference section if desired).  I'll add info about expected worker shutdowns and the safety shutdown to the AWS provisioner section.